### PR TITLE
Updated formatting in HTML files

### PR DESCRIPTION
**Pull request by skymandr**
I found the UI code very hard to navigate, so I reformatted the HTML
code.  There should be no functional changes.

### CODE REVIEW

File: scanomatic/ui_server_data/access_restricted.html [hunk 1]
```diff
@@ -10,9 +10,14 @@
 <div id='cont'>
 	<div class='section'>
 		<h1>Access Restricted</h1>
-		<p>You have attempted to access information that you shouldn't, this has been reported.</p>
-		<p>If you consider the previous statement not true, make sure that the admin runs Scan-o-Matic with
-		appropriate access settings.</p>
+		<p>
+            You have attempted to access information that you shouldn't, this
+            has been reported.
+        </p>
```
Comment (gdetrez) [inline scanomatic/ui_server_data/access_restricted.html]: Strange indentation

File: scanomatic/ui_server_data/access_restricted.html [hunk 2]
```diff
@@ -10,9 +10,14 @@
...
+		<p>
+            If you consider the previous statement not true, make sure that the
+            admin runs Scan-o-Matic with appropriate access settings.
+        </p>
```
Comment (gdetrez) [inline scanomatic/ui_server_data/access_restricted.html]: Strange indentation

File: scanomatic/ui_server_data/access_restricted.html (hunk 1 above)
Comment (skymandr) [reply scanomatic/ui_server_data/access_restricted.html]: Mixing tabs and spaces. Fixed it.

File: scanomatic/ui_server_data/access_restricted.html (hunk 2 above)
Comment (skymandr) [reply scanomatic/ui_server_data/access_restricted.html]: Same here.

**Pull request merged**